ntp-proto/src/packet/v5/mod.rs

```diff
@@ -118,9 +121,18 @@ impl NtpServerCookie {
 pub struct NtpClientCookie(pub [u8; 8]);
 
 impl NtpClientCookie {
+    fn new_random() -> Self {
+        // TODO does this match entropy handling of the rest of the system?
+        Self(random())
```

Comment:
it is kind of unfortunate for testing (potentially for reproducing fuzzing errors too)